utils/install_actor_deps.py

```diff
@@ -30,7 +30,7 @@ def install(path):
 
 
 def install_actor_deps(actor, directory):
-    for root, dirs, files in os.walk(directory):
+    for root, dirs, _ in os.walk(directory):
```

Comment:
the use of `_` in our project is kind of prohibited because of conflict with the `_` function. use the dummy_files instead for example.

Reply:
Could you please elaborate on this. I see the `_` variable inside /home/azhukov/Dropbox/code/leapp/leapp/utils/i18n.py, which is used only once in leapp/snactor/__init__.py
```
from leapp.utils.i18n import _  # noqa; pylint: disable=redefined-builtin
```
Also `_` is already used in 7 places inside this repo and it is a typical python convention to skip the variable when unpacking

Reply:
use of dummy prefix/suffix is kind of typical as well, right because of this. It is possible in future that we will need to deal with it, it's better to not use it. Originally it was mentioned in our guidelines as well. Not sure whether it is there as well, anyway, do not use it. As well, it could happen in future that someone will need to start to use such dummy variable. So that's good reason why to put there something more clear about what's inside.

Existence of `_` in our project doesn't mean we want to start to use it everywhere. Maybe it's just a mistake that reviewers did and passed these variables, but they shouldn't.

Reply:
actually, this is one of things, that pre-commit hooks could detect ;-) but again, people should be guided how to ignore it for bundled libraries.